rpc/client/rpc_test.go

```diff
@@ -494,6 +494,13 @@ func TestTxSearch(t *testing.T) {
 			t.Fatal("expected a lot of transactions")
 		}
 
+		// query using an index key
+		result, err = c.TxSearch("app.index_key='index is working'", false, 1, 30, "asc")
```

Comment:
It would be nice to test the negative as well, i.e. that a field with `Index: false` does not return results.

Reply:
added a test case for that.